tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -845,6 +845,32 @@ namespace Tensile
                 }
             };
 
+            struct GsuLessEqualOne : public Predicate_CRTP<GsuLessEqualOne, ContractionProblemGemm>
```

Comment:
only reject when problem's amaxD is enabled and GSU > 1

Reply:
done, Contractions.py and ContrationProblemPredicates.hpp are updated